1-Draft/RFCXXXX-Policy.md

```diff
@@ -0,0 +1,243 @@
+| File -> Computer-Wide       | %ProgramFiles%\PowerShell\powershell.config.json     | /opt/Microsoft/powershell/powershell.config.json     |
+| GPO -> User Config          | HKCU\Software\PowerShellCore                         | See `Comment A` below                                |
+| GPO -> Computer Config      | HKLM\Software\PowerShellCore                         | /etc/powershell.config.json                          |
+
+### Configuration settings
+
+A set of configuration settings in GPO scheme and file scheme for policy settings and regular settings is the same. This allows to discover and configure settings in the simplest and fastest way.
+
+#### Registry keys and settings
+
+| Key                              | SubKey                      | Option                             | Type   | Precedence          |
+|----------------------------------|-----------------------------|------------------------------------|--------|---------------------|
+| Software\Policies\PowerShellCore | -                           | -                                  |        |                     |
+| Software\PowerShellCore          | -                           | -                                  |        |                     |
+|                                  |                             | ExecutionPolicy                    | String | Computer, Then User |
+|                                  | ConsoleSessionConfiguration | EnableConsoleSessionConfiguration  | DWORD  | User, then Computer |
+|                                  | ConsoleSessionConfiguration | ConsoleSessionConfigurationName    | String | User, then Computer |
+|                                  | ModuleLogging               | EnableModuleLogging                | DWORD  | Computer, Then User |
+|                                  | ModuleLogging               | ModuleNames                        | String | Computer, Then User |
+|                                  | ProtectedEventLogging       | EncryptionCertificate              | DWORD  | Computer Wide       |
+|                                  | ScriptBlockLogging          | EnableScriptBlockInvocationLogging | DWORD  | Computer, Then User |
+|                                  | ScriptBlockLogging          | EnableScriptBlockLogging           | DWORD  | Computer, Then User |
+|                                  | Transcription               | EnableTranscripting                | DWORD  | Computer, Then User |
+|                                  | Transcription               | EnableInvocationHeader             | DWORD  | Computer, Then User |
+|                                  | Transcription               | OutputDirectory                    | String | Computer, Then User |
+|                                  | UpdatableHelp               | DefaultSourcePath                  | String | Computer Wide       |
```

Comment:
This table is a bit confusing to me. Do `Software\Policies\PowerShellCore` and `Software\PowerShellCore` have the same sub keys?

Reply:
There are no current keys listed for policies.  I'll update the table as the current comment says.